plugins/kotlin/formatter/src/org/jetbrains/kotlin/idea/formatter/trailingComma/TrailingCommaState.kt

```diff
@@ -65,7 +65,7 @@ private fun isMultiline(ktElement: KtElement): Boolean = when {
     )
 
     ktElement is KtWhenEntry -> ktElement.isMultiline(
-        startOffsetGetter = { startOffset },
+        startOffsetGetter = { conditions.lastOrNull()?.endOffset },
```

Comment:
Enforce trailing comma only in the case there is line break between last condition and arrow.
I'm not sure this fix is correct but iy works.

Reply:
`startOffset` is correct here. 
Was:
```kotlin
fun t() {
    when (true) {
        <getter scope>true, false,
        true, false </getter scope>-> {}
    }
}
```
After the change:
```kotlin
fun t() {
    when (true) {
        true, false,
        true, false<getter scope> </getter scope>-> {}
    }
}
```

Reply:
Dropped this change